src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -736,17 +724,6 @@ private static CompletionList RemovePreselection(CompletionList completionList)
             return completionList;
         }
 
-        // C# keywords were previously provided by snippets, but as of now C# LSP doesn't provide snippets.
-        // We're providing these for now to improve the user experience (not having to ESC out of completions to finish),
-        // but once C# starts providing them their completion will be offered instead, at which point we should be able to remove this step.
-        private static CompletionList IncludeCSharpKeywords(CompletionList completionList)
-        {
-            var newList = completionList.Items.Union(s_keywordCompletionItems, CompletionItemComparer.Instance);
```

Comment:
Should be able to nuke a few more things in this class:

- IsRazorCompilerBugWithCSharpKeywords
- s_keywordCompletionItems
- WordSpanMatchesCSharpPolyfills
- s_keywords